CMakeLists.txt

```diff
@@ -780,11 +780,15 @@ if (CMAKE_Swift_COMPILER)
 else()
   message(STATUS "Swift Compiler (None).")
 endif()
-if(SWIFT_PATH_TO_CMARK_BUILD)
-  execute_process(COMMAND ${SWIFT_PATH_TO_CMARK_BUILD}/src/cmark --version
-    OUTPUT_VARIABLE _CMARK_VERSION
-    OUTPUT_STRIP_TRAILING_WHITESPACE)
-  message(STATUS "CMark Version: ${_CMARK_VERSION}")
+if(SWIFT_INCLUDE_TOOLS)
+  if(SWIFT_PATH_TO_CMARK_BUILD)
```

Comment:
The CMark changes are all the same as https://github.com/apple/swift/pull/70791except for this part, where `SWIFT_PATH_TO_CMARK_BUILD` is pulled into `SWIFT_INCLUDE_TOOLS`. Should that be done in that PR (I assume you'll rebase this one after that one is in?

Reply:
Yeah, I intend to rebase this after the initial switch over is done.